README.md

```diff
@@ -57,10 +57,87 @@ AirbrakesV2/
 │   ├── ...
 ├── main.py [main file used to run on the rocket]
 ├── constants.py [file for constants used in the project]
-├── pyproject.toml [configuration file for the project]
+├── pyproject.toml [confipip install mkdocs-mermaid2-pluginguration file for the project]
```

Comment:
```suggestion
├── pyproject.toml [configuration file for the project]
```

revert this change. The mermaid diagram is only meant to be seen on github so it's okay not having that dependency

Reply:
removed it in the latest change!